### ISSUE TITLE
Runtime checks for samples causes a lot of false positives for short runtime shows

### ISSUE THREAD

**Issue by Tenno6** [Sonarr/Sonarr#7086]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Currently media that include episodes with low runtimes wont import due to being detected as a sample. 

### Expected Behavior

Either take into account the type of show (cartoons with short episodes ex. Looney Tunes) to disable sample detection or either a global sample runtime detection setting or individual setting per show. 

### Steps To Reproduce

1. Fetch episode with low runtime
2. Sonarr fails to import 

### Environment

```markdown
- OS: unraid
- Sonarr: 4.0.8.1874
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: N/A
- Database: Sqlite 3.45.3
```

### What branch are you running?

Main

### Trace Logs?

N/A

**Comment by Tenno6** [comment]
2024-08-12 08:46:34.7|Debug|DetectSample|[/downloads/usenet/completed/tv/Looney.Tunes.S1937E36.September.in.the.Rain.DVDRip.AC3.x264/Looney.Tunes.S1937E36.September.in.the.Rain.DVDRip.AC3.x264.mkv] appears to be a sample. Runtime: 00:04:39.2790000 seconds. Expected at least: 300 seconds

**Comment by bakerboy448** [comment]
Impact/frequency seems like it'd be isolated to one or two shows that release episodes in a per file per short format.....most or almost all of these types of releases will end up as several shorts together in one file.

Seems like work to resolve what manual import for the handful of shows / releases would occur
